src/pages/admin/admin.tsx

```diff
@@ -156,20 +172,34 @@ const Admin: React.FC = () => {
     const dbSnap = await getDoc(dbRef);
     if (dbSnap.exists()) {
       alert(`${newOpenTeam} already exists!`);
-    }
-    else {
+    } else {
+      let channelId = '';
+      const chanIdRef = doc(FirestoreDB, 'channelIDs', 'channelIDs');
+      const chanIdSnap = await getDoc(chanIdRef);
+      const chanIdData: channelIDS[] = chanIdSnap.data().channelData;
+      for (let i = 0; i < chanIdData.length; i++) {
```

Comment:
I think we should change the database structure. Instead of having everything in one doc, we should have a list of docs in a collection so that we can use "where" query instead of for loops.